contracts/MedicalRecord.sol

```diff
@@ -0,0 +1,88 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+contract MedicalRecord {
+    struct medicalRecordInfo {
+      uint totalApproved;
+      uint totalPending;
+      uint totalRejected;
+    }
+
+    struct recordData {
+      string ipfsHash;
+      status recordStatus;
+    }
+
+    enum status {
+        PENDING,
+        APPROVED,
+        REJECTED
+    }
+
+    mapping(address => medicalRecordInfo) public userToMedicalRecords;
+    mapping(address => recordData[]) public userToData;
+    mapping(string => recordData) public ipfsToRecordData;
+
+    event setStateToMedicalRecord(status _setState, string _ipfsHash);
+    event addNewMedicalRecord(string _ipfsHash, address _userAdress, status _addNew);
+
+    function setState(address _userAddress, string memory _ipfsHash,status _statusRecord) public {
+      require(_statusRecord != status.PENDING, "This state is already in this medical record");
```

Comment:
This medical record is already in this state*

Reply:
ok